src/test/java/testtype/CVSB_902/LECTestType_2579.java

```diff
@@ -27,6 +29,8 @@
     @Steps
     TestTypeDetailsSteps testTypeDetailsSteps;
 
+    @WithTag("In_test")
+    @Ignore("Rendered obsolete by CVSB-7936")
```

Comment:
Why using In_test tag here since test is anyway ignored?

Reply:
fixed